### ISSUE TITLE
Bug: ELRS Backpack control on Lowband

### ISSUE THREAD

**Issue by zandorsp** [hd-zero/hdzero-goggle#378]
To reproduce:
- Use a Expresslrs backpack flashed TX (used TX16S) ver 1.4.1 and goggles flashed with 1.4.1 too
- Select raceband and use the lua script to change the channels R1 to R8, and see work nicely, changing the channel on goggles  and vtx at same time.
- Now select lowband on goggles and try the same on channels L1 to L8.... The vtx change the channel ok, but the goggles doesn't change.

Others functions of the backpack works nicely, like the DVR control.

**Comment by zandorsp** [comment]
One question: Why is necessary to have a menu with Raceband or Lowband for selection? Why not a Lowband on/off and select the channels L1-L8,R1-R8,F1,F2,F4,E1 freely ?

**Comment by zandorsp** [comment]
> * Did you unlock lowband for your vtx?
> * In fact, most people use raceband, and for them lowband is redundant.

Yes, I unlocked lowband on vtx, and the vtx is working well on lowband, this affect only the Goggles and backpack

**Comment by pitts-mo** [comment]
Is there a technical issue that prevents both the normal channels and low
band channels from being visible in menu at same time?

It makes more sense to have them always visibly and just change the menu
toggle to grey them out and skip over them when selecting/scanning.

**Comment by zandorsp** [comment]
I think a lowband unlock on goggles option make sense... And if unlocked lowband and normal channels is selectable freely, If locked only channels R1-R8, E1, F1,F2 and F4 selectable.

**Comment by ligenxxxx** [comment]
Displaying all channels (20) at the same time will make the UI look very bloated, and I don't like complicated things. And scanning all channels at the same time will be more time-consuming than before. I did this first, but the experience was not good.

**Comment by pitts-mo** [comment]
Agree it sounds bloated, unneeded, and undesirable for most users.

How about we try to support all channels at once, only
display/select/search from one band at a time, default to display/search
the band current channel exists in, move band toggle from sources page and
add a hint for quick toggle/transition feature into the channels display
page.

Maybe?
This way all channels feel more accessable for users and the ability to
switch quickly will likely improve user education..
-p



Just

**Comment by zandorsp** [comment]
I have a sugestion for the UI on scan now menu:
![highband](https://github.com/hd-zero/hdzero-goggle/assets/53715440/530e41ba-476a-412e-9e5a-9d36280786e1)
![imd6](https://github.com/hd-zero/hdzero-goggle/assets/53715440/5feb7d39-3abe-4e64-8f4c-e0ddf1df9824)
![japan](https://github.com/hd-zero/hdzero-goggle/assets/53715440/39c87e15-f6bf-448d-86b5-d7a07679f99a)

On live view all 20 channels available on dial (L1-L8,R1-R8,E1,F1,F2,F4)
and a lowband unlock option

**Comment by lesharodin** [comment]
We started flying lowband more, when we have 8 pilots in the air.
Its so frustrating going to menu and change from race to low. Please make option so all freqs available in live view

**Comment by zandorsp** [comment]
> We started flying lowband more, when we have 8 pilots in the air. Its so frustrating going to menu and change from race to low. Please make option so all freqs available in live view


+1

**Comment by KidCe** [comment]
This is so important for EU Racers!

At Aircrasher events (e.g. FAI Worldcup Aichtal or MultiGP European Championship) Lowband is a necessity for ALL pilots to have Lowband accessable quickly. It can always happen that you get placed on a different channel shortly before the race. 

UI Bloat is not a argument in my opinion, because most people i know cant even use google UI because of very bad Scroll wheel. Everyone uses ELRS Backpack for channel change and never thouch their goggle.

If you need to switch it to lowband via the UI, every one with Scrollwheel problems is at risk that they get placed on Lowband Channels and they will have to try for minutes to switch to lowband while all the other pilots have to wait.

ask me how i know.

Or worst case you get disqualified form that race because you cant switch in time.

Either Lowband needs to be added to google backpack switch, or a reliable Scroll wheel fix for everyone needs to be found.

**Comment by bob9** [comment]
#614  I added this PR yesterday that allows you to switch to and from low band via expresslrs backpack. I'll built it and test it and it works.
